flycheck.el

```diff
@@ -6925,6 +6925,13 @@ See URL `http://www.erlang.org/'."
 (flycheck-define-checker erlang-rebar3
   "An Erlang syntax checker using the rebar3 build tool.
 
+This flychecker compiles the entire rebar3 project, which
+compensates for contextual errors that the regular erlang checker
+wouldn't be aware of. One such example is the erlang -include
+directive. Rebar3 is aware of the context for those includes,
+where the standard erlang checker is not. This checker should be
+used instead of the erlang checker for all rebar3 projects
```

Comment:
Thanks but I didn't mean to clarify that in the docstring, but rather write a comment on the Github issue :blush:  Please remove this rather lengthy paragraph again.